src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeActionService.java

```diff
@@ -135,4 +135,8 @@ private boolean isStale(Envelope envelope) {
             .orElseThrow(); // no events for the envelope is normally impossible
         return between(lastEventTimeStamp, now()).toHours() > notificationTimeoutHr;
     }
+
+    private String getErrorMessage(UUID id, String msg) {
+        return "Envelope with id " + id + " " + msg;
```

Comment:
may be string builder?

Reply:
not necessary anymore: https://dzone.com/articles/string-concatenation-performacne-improvement-in-ja#:~:text=As%20you%20can%20see%20in,always%20suggested%20to%20use%20StringBuilder.